Spec/RealtimeClientConnection.swift

```diff
@@ -2649,6 +2649,91 @@ class RealtimeClientConnection: QuickSpec {
                         }
                     }
                 }
+                
+                // RTN15g
+                context("when connection (ttl + idle interval) period has passed since last activity") {
+                    
+                    it("uses a new connection") {
+                        let options = AblyTests.commonAppSetup()
+                        var client: ARTRealtime!
+                        var connectionId = ""
+                        let customTtlInterval: TimeInterval = 1.0
+                        let customIdleInterval: TimeInterval = 1.0
+                        // We want this to be > than the sum of the two above
+                        let reconnectAfterInterval: TimeInterval = 5.0
+                        client = AblyTests.newRealtime(options)
+                        client.connect()
```

Comment:
@funkyboy I usually add a `defer` closing the current connection. The connection will remain active. Something like `defer { client.close() }`.

Reply:
@ricardopereira done https://github.com/ably/ably-ios/pull/719/commits/8565908636c1c3b7c123abbe7520870eb5661c15